buildh/cli.py

```diff
@@ -174,52 +215,42 @@ def main():
 
     # Check if the lipid topology match the the structure.
     if not lipids.check_topology(universe_woH, dic_lipid):
-        sys.exit(f"The topology chosen does not match the structure provided in {args.coord}")
+        raise BuildHError(f"The topology chosen does not match the structure provided in {coord_file}")
 
     # Check if atoms name in the def file are present in the structure.
     atoms_name = [heavy_atom for (heavy_atom, _) in dic_atname2genericname.keys()]
     if not utils.check_def_file(universe_woH, dic_lipid['resname'], atoms_name):
-        sys.exit(f"Atoms defined in {args.defop} are missing in the structure {args.coord}.")
+        raise BuildHError(f"Atoms defined in {def_file} are missing in the structure {coord_file}.")
 
     # Check the def file and the topology are coherent.
     if not utils.check_def_topol_consistency(dic_Cname2Hnames, dic_lipid):
-        sys.exit(f"Atoms defined in {args.defop} are not consistent with topology chosen.")
+        raise BuildHError(f"Atoms defined in {def_file} are not consistent with topology chosen.")
```

Comment:
I guess it should be *are not consistent with the chosen topology*.

Reply:
> Good idea to create a BuildHError exception.
> To me your PR looks fine.
Great!
> One question, I saw you removed the pickle stuff. It has nothing to do with using buildH as a module, this is just cleaning up, right?
Yes indeed. I thought it was not useful anymore.